recipe/meta.yaml

```diff
@@ -11,8 +11,8 @@ source:
   sha256: e586fb492459d6ab80f442929b622d1567d2dc77bcffcdf49ac5146567b22944
 
 build:
-  number: 2
-  skip: true  # [py<36]
+  number: 3
+  skip: true  # [py<36] [python_impl == 'pypy']
```

Comment:
Not sure if it's `[py<36 or python_impl == 'pypy']`?

Reply:
From the tests that were run, it looks like both work? Anyways I created the PR from a branch, not a fork, and need to create a new one.

Reply:
:+1: